src/bindist/autoemu.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
 rem generic autoexec.bat for DOSEMU + any DOS
 rem must be edited in most circumstances
-path c:\dos;c:\windows\command;e:\dosemu
-lredir d: linux\fs\${DOSEMU_LIB_DIR}/freedos ro
+set PATH=c:\windows\command;e:\dosemu

```

Comment:
I really don't understand how this could work instead of using EXIST to build up the PATH. I cannot control which environment variables will be passed into dosemu and I don't really see what I could pass to add to the PATH which could be useful.